tools/LibraryVersionsGenerator/Models/PackageVersion.cs

```diff
@@ -5,10 +5,13 @@ namespace LibraryVersionsGenerator.Models;
 
 internal class PackageVersion
 {
-    public PackageVersion(string version)
+    public PackageVersion(string version, params string[] supportedFrameworks)
```

Comment:
I think using a second constructor removes all of that `Array.Empty<string>()` bloat.

Reply:
38319538bc8a57a994866d45b4f5f1bf0cd2c663